guava-tests/test/com/google/common/math/QuantilesTest.java

```diff
@@ -701,6 +701,15 @@ public void testScale_index_computeInPlace_empty() {
     }
   }
 
+  public void testEmpty_indexes(){
```

Comment:
Other test cases in this block use a common naming convention with words `Scale`, `indexes`, `empty` in that order. I suggest following that convention.